### ISSUE TITLE
(Request) Typedef for type aliases

### ISSUE THREAD

**Issue by lucasmciruzzi** [joelday/vscode-docthis#43]
It will be great to have documentation for [type aliases](https://www.typescriptlang.org/docs/handbook/advanced-types.html#type-aliases) trough JSDoc's [@typedef](http://usejsdoc.org/tags-typedef.html): 
```typescript
/**
 * Description.
 *
 * @typedef {(string|number)} stringOrNumber
 */
type stringOrNumber = string | number;
```
```typescript
/**
 * Description.
 *
 * @export
 * @typedef {(string|number)} exportedStringOrNumber
 */
export type exportedStringOrNumber = string | number;
```

**Comment by lucasmciruzzi** [comment]
I think the solution will be something like adding this to the [_documentNode](https://github.com/joelday/vscode-docthis/blob/master/src/documenter.ts#L233) method in the [Documenter class](https://github.com/joelday/vscode-docthis/blob/master/src/documenter.ts):

```typescript
case ts.SyntaxKind.TypeAliasDeclaration:
  this._emitTypeAliasDeclaration(sb, <ts.TypeAliasDeclaration>node);
  break;
```

**Comment by TheOpenDevProject** [comment]
This is `TypeScript` syntax right?

**Comment by lucasmciruzzi** [comment]
@TheOpenDevProject yup. If you click on "type aliases" you'll get to the TypeScript specification :)

**Comment by TheOpenDevProject** [comment]
Ok ill have a look tomorrow when im hacking out some other code for the project.

**Comment by joelday** [comment]
Please see https://github.com/joelday/vscode-docthis/issues/196! Thank you!
